backend/app/utils/Ocr.scala

```diff
@@ -41,19 +64,19 @@ object Ocr {
         stdout.mkString("\n")
 
       case _ =>
-        throw new OcrSubprocessCrashedException(exitCode, stderr.mkString("\n"))
+        throw new OcrSubprocessCrashedException(exitCode, stderr.getOutput)
     }
   }
 
   // TODO MRB: allow OcrMyPdf to read DPI if set in metadata
   // OCRmyPDF is a wrapper for Tesseract that we use to overlay the OCR as a text layer in the resulting PDF
-  def invokeOcrMyPdf(lang: String, inputFileName: String, dpi: Option[Int], stderr: mutable.Buffer[String], tmpDir: Path): Path = {
+  def invokeOcrMyPdf(lang: String, inputFileName: String, dpi: Option[Int], stderr: OcrStderrLogger, tmpDir: Path): Path = {
     val tempFile = Files.createTempFile("ocrmypdf", ".pdf")
     val cmd = s"ocrmypdf --redo-ocr -l $lang ${dpi.map(dpi => s"--image-dpi $dpi").getOrElse("")} ${inputFileName} ${tempFile.toAbsolutePath}"
 
     val stdout = mutable.Buffer.empty[String]
     val process = Process(cmd, cwd = None, extraEnv = "TMPDIR" -> tmpDir.toAbsolutePath.toString)
-    val exitCode = process.!(ProcessLogger(stdout.append(_), stderr.append(_)))
+    val exitCode = process.!(ProcessLogger(stdout.append(_), stderr.append))
```

Comment:
is the removal of the second `_` significant or just syntactic sweetening?

Reply:
Just sweetening, without it IntelliJ highlighted as possible to remove. I'm not sure why it was required before but not now this component is a class? I think the underscore is not necessary at all in Scala 3: https://dotty.epfl.ch/docs/reference/changed-features/eta-expansion.html